public/src/components/channelManagement/bannerDesigns/HeadlineSizeEditor.tsx

```diff
@@ -0,0 +1,56 @@
+import { FormControl, FormControlLabel, RadioGroup, Radio } from '@mui/material';
+import React from 'react';
+import { FontSize } from '../../../models/bannerDesign';
+
+interface Props {
+  headerSize?: FontSize;
+  isDisabled: boolean;
+  onValidationChange: (fieldName: string, isValid: boolean) => void;
```

Comment:
I don't think we need validation in this component as it uses a radio group

Reply:
Good point - thanks!